### PR TITLE
[OJS][main] #9197 Craft OA Backoffice - AccessKey Infrastructure Removal

### PR DESCRIPTION
**Pull request by defstat**

### CODE REVIEW

**Review by asmecher — CHANGES_REQUESTED** [review]
It looks like you're missing changes to `dbscripts/xml/upgrade.xml` and `dbscripts/xml/install.xml` to upgrade/install.

File: pages/reviewer/ReviewerHandler.php [hunk 1]
```diff
@@ -64,79 +58,35 @@ public function authorize($request, &$args, $roleAssignments)
...
+                    PKPInvitationHandler::REPLY_OP_ACCEPT,
+                    null,
+                    [
+                        'key' => $accessKeyCode,
+                    ]
+                );
+
+                header('HTTP/1.1 301 Moved Permanently');
+                $request->redirectUrl($url);
+                return true;
```
Comment (asmecher) [inline pages/reviewer/ReviewerHandler.php:77]: This is dead code; `PKPRequest::redirect` [stops execution](https://github.com/pkp/pkp-lib/blob/93e4a76fe2018e05d0830b677c43294017fb8724/classes/core/PKPRequest.php#L128).

Comment (defstat) [reply pages/reviewer/ReviewerHandler.php:77]: Sure, removed

File: pages/reviewer/ReviewerHandler.php [hunk 2]
```diff
@@ -64,79 +59,41 @@ public function authorize($request, &$args, $roleAssignments)
...
+            $accessKeyCode = $request->getUserVar('key');
+            if ($accessKeyCode) {
+                $keyHash = md5($accessKeyCode);
+
+                $invitation = Repo::invitation()->getBOByKeyHash($keyHash);
+
+                if (isset($invitation)) {
+                    $invitation->acceptHandle();
+                }
+                // $url = $request->getDispatcher()->url(
```
Comment (asmecher) [inline pages/reviewer/ReviewerHandler.php:71]: Is this area of code functioning? If so, please remove the commented code.

**Pull request merged**